python/requirements.txt

```diff
@@ -1,16 +1,20 @@
-# coap
-aiocoap==0.3
-cbor2==4.0.0
-
-# deployment
-# packaging enables pip install -e
+# Deployment
+# Packaging enables pip install -e
 packaging==16.8
 
-# rest
+# Common - REST interface
 aiohttp==2.3.6
 aiohttp_cors==0.5.3
 cchardet==2.1.1
 requests==2.18.4
 
-# sensortag
+# PostreSQL Storage Plugin
+psycopg2
```

Comment:
@izoratti fyi, we had a practice for new dependency along with its version (stable and latest one)
anyways fixed by @MarkRiddoch in https://github.com/foglamp/FogLAMP/pull/414/files#diff-b1432fb9380829bd394612c5565750abR3